internal/nullable/value.go

```diff
@@ -16,10 +16,17 @@ type Value[T any] struct {
 	Value T
 }
 
-// FromString[T any] represents a parser function. It is used to avoid
+func New[T nullable](v T, set bool) Value[T] {
+	return Value[T]{
+		Set:   set,
+		Value: v,
+	}
+}
+
+// Parser[T any] represents a parser function. It is used to avoid
 // a single parser for all nullables for improved readability and performance.
-// FromString should always return Nullable with Set=true, error otherwise.
-type FromString[T any] func(string) (Value[T], error)
+// Parser should always return Nullable with Set=true, error otherwise.
+type Parser[T any] func(string) (Value[T], error)
```

Comment:
T can be `any` or it should `nulllable` ?

Reply:
Should be `nullable`, fixed.